packages/create-cloudflare/src/cli.ts

```diff
@@ -30,6 +32,39 @@ const WRANGLER_DEFAULTS = {
 export const main = async (argv: string[]) => {
 	const args = await parseArgs(argv);
 
+	// Print a newline
+	logRaw("");
+
+	if (args.autoUpdate && (await isUpdateAvailable())) {
+		await runLatest();
+	} else {
+		await runCli(args);
+	}
+};
+
+const isUpdateAvailable = async () => {
+	if (process.env.VITEST) {
+		return false;
+	}
+
+	// Use a spinner when running this check since it may take some time
+	const s = spinner(spinnerFrames.vertical);
+	s.start("Checking for newer version");
```

Comment:
```suggestion
	s.start("Checking for updates");
```
? 🤔

Reply:
Updates feels weird, how about "Checking if a newer version is available"?

Reply:
Nah don't worry checking for newer version is fine... I just feel that it sounded a bit specific, but I'm totally fine with it if you prefer it 🙂👍